l5x/test.L5X

```diff
@@ -410,6 +415,26 @@ BHAIG29GI]]>
 </Structure>
 </Data>
 </Tag>
+<Tag Name="multiArray" TagType="Base" DataType="INT" Dimensions="2 4" Radix="Decimal" Constant="false" ExternalAccess="Read/Write">
+<Description>
+<![CDATA[Demo of multi-dimensional array]]>
+</Description>
+<Data Format="L5K">
+<![CDATA[[5,10,20,99,12,26,74,194993]]]>
+</Data>
+<Data Format="Decorated">
+<Array DataType="INT" Dimensions="2,4" Radix="Decimal">
```

Comment:
Another inference question: since we have the multidimensional indices, could the Dimensions value be inferred from that, or, is the idea more that we can validate the correct supplied elements?

Reply:
Probably true. But I don't know if the supplied data is always provided. Validating would probably make sense. (There's also duplicate data, since the `CDATA` field contains the same information as the XML in the array, so that could also be compared for validation.)